frontend/src/v5/ui/controls/dragAndDrop/dragAndDrop.component.tsx

```diff
@@ -15,43 +15,22 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { useState } from 'react';
-import { Container, DashedContainer, DropArea } from './dragAndDrop.styles';
+import { DashedContainer, DropArea } from './dragAndDrop.styles';
+import { useDropzone } from 'react-dropzone';
 
 interface IDragAndDrop {
-	hidden?: boolean;
 	onDrop: (files) => void,
 	className?: string;
 	children?: any;
 	accept?: string;
 }
-export const DragAndDrop = ({ children, onDrop, hidden = false, ...props }: IDragAndDrop) => {
-	const [dragOverlay, setDragOverlay] = useState(false);
-
-	const handleDragEnter = (e) => {
-		if (e.dataTransfer.items.length > 0) {
-			setDragOverlay(true);
-		}
-	};
-
-	const handleDragLeave = () => {
-		setDragOverlay(false);
-	};
-
-	const handleDrop = (files) => {
-		onDrop(files);
-		setDragOverlay(false);
-	};
+export const DragAndDrop = ({ children, onDrop, ...props }: IDragAndDrop) => {
```

Comment:
In staging, dropping a file with an invalid (not supported) extension on the container revision upload modal does nothing. Now, instead, it creates a new revision entry
![image](https://github.com/user-attachments/assets/1710d64b-bf18-4795-b71e-52e12f1bf2f3)